### DISCUSSION TITLE
Comment / Popup Feature in V3 using UMD?

### DISCUSSION THREAD

**Discussion by mrvnklm** [annotorious/annotorious#470] [Q&A]
Hi there, I am wondering if the comment or at least the popup feature is still available in V3 especially when using the UMD without any build tools? I did not find anything about it in the v3 docs anymore.

**Comment by rsimon** [comment]
At the moment: sorry, no. The new version doesn't come with a built-in popup. For now, my recommendation is to keep using v2.7.

I am planning to build a basic comment popup for the vanilla JS 3.0 version (incl. UMD script import) eventually. But, to be honest, it's not the highest priority on the list, so I can't say when that will happen. (It all depends on whether/when I'll need one for one of my own projects at some point, I guess.)

**Reply by mrvnklm** [reply]
Thank you for your fast response. I will figure something else out for now then, but as I am using some basic alpineJS it would be nice to at least have the basic popup functionality so anybody could implement their own logic at least :)
